### ISSUE TITLE
File not found 

### ISSUE THREAD

**Issue by subm** [mjnaderi/Sharif-Judge#35]
When I make a submission, It shows file not found with a blue status

**Comment by aidarbek** [comment]
I have the same error. Mac OS X Yosemite
![2015-05-31 02 37 25](https://cloud.githubusercontent.com/assets/5136677/7899256/ffc2f076-073d-11e5-9a4e-a038511e3c67.png)

**Comment by mjnaderi** [comment]
Ensure that `tester` and `assignments` directories are writable by the user running PHP.

**Comment by aidarbek** [comment]
@mjnaderi No, it did not help. For making sure, i made chmod 777 for all subdirectories, but it did not make sense. Here is a log file, found in `assignments/assignment_1/p1/aidarbek`

![2015-06-01 11 27 38](https://cloud.githubusercontent.com/assets/5136677/7906738/93aeb798-0851-11e5-8162-d2a5aa5a7ca1.png)

**Comment by aidarbek** [comment]
@mjnaderi it seems like it successfully checked solution, but I do not know why does it write 'File not found'.

**Comment by ghost** [comment]
I encounter the same issue here, and could not solve it yet.

**Comment by linhtlthp** [comment]
Setting error!

2015-06-03 0:44 GMT+07:00, nour-ramadan notifications@github.com:

**Comment by guangyu-ryan** [comment]
@aidarbek I faced the same problem, fortunately, I solved it. It seems that the user's code is compiled successfully, but the output don't match the answer in the output file.

**Comment by jimgao1** [comment]
Same problem here. Seemed that the judge cannot access the log file either. However, I can tell that the compilation succeed by looking at the log file in the assignments folder.

**Comment by atiabjobayer** [comment]
I have solved the problem. You guys probably kept the problem description in a folder and then compressed that folder. For that, this happens.

You have to compress only the folders like "p1,p2,....." .

I think this will help.

**Comment by jimgao1** [comment]
Thank you @atiabjobayer. This is how I solved the issues. I realized that I named my input/output folders wrong names. Make sure that you only compress the problems folders, and name the i/o folders in and out.

**Comment by ehsanasgari** [comment]
I am trying to use Sharif Judge for a simple string manipulation problem. The problem is to permute a word in specific way. So I made test files (zip file:in>input1.txt (content: book)  out>output.txt (content:ookb)). I am getting the same error. May I ask how does a python submission in Sharif Judge look like? Shall we use sys.argv?

msg = ' '.join(sys.argv[1::])
print (scramble(msg))

**Comment by alphabbet** [comment]
@aidarbek hello sir,did you solved this problem?
